go/sentry/api/api.go

```diff
@@ -13,9 +14,22 @@ type SentryAddresses struct {
 	Committee []node.CommitteeAddress
 }
 
+// UpstreamTLSCertificates contains the upstream TLS certificates.
+type UpstreamTLSCertificates struct {
+	Certificate     *tls.Certificate
+	NextCertificate *tls.Certificate
+}
+
 // Backend is a sentry backend implementation.
 type Backend interface {
 	// Get addresses returns the list of consensus and committee addresses of
 	// the sentry node.
 	GetAddresses(context.Context) (*SentryAddresses, error)
+
+	// SetUpstreamTLSCertificates notifies the sentry node of the new
+	// TLS certificates used by its upstream node.
+	SetUpstreamTLSCertificates(context.Context, *tls.Certificate, *tls.Certificate) error
```

Comment:
Use `*UpstreamTLSCertificates` as type here (or use the alternative `[][]byte` mentioned above).